base_layer/wallet/tests/utxo_scanner.rs

```diff
@@ -288,7 +288,7 @@ async fn test_utxo_scanner_recovery() {
     let mut test_interface = setup(UtxoScannerMode::Recovery, None, None, None).await;
 
     let cipher_seed = CipherSeed::new();
-    let birthday_epoch_time = u64::from(cipher_seed.birthday() - 2) * 60 * 60 * 24;
+    let birthday_epoch_time = get_birthday_from_unix_epoch(cipher_seed.birthday(), 14u16);
```

Comment:
Nit: recommend a comment clearly indicating the units, since we switch between seconds and days.

Reply:
Definitely !